doc/changes/0.19.inc

```diff
@@ -27,7 +27,7 @@ Changelog
 
 - Add :func:`mne.channels.read_dig_polhemus_isotrak` and :func:`mne.channels.read_polhemus_fastscan` to read Polhemus data by `Joan Massich`_
 
-- Add :func:`mne.channels.read_dig_captrack` to read BrainVision CapTrak (BVCT) digitization coordinate files by `Stefan Appelhoff`_ and `Joan Massich`_
+- Add :func:`mne.channels.read_dig_captrak` to read BrainVision CapTrak (BVCT) digitization coordinate files by `Stefan Appelhoff`_ and `Joan Massich`_
```

Comment:
Usually we just change to using double-backticks (change to code rather than link) and otherwise preserve the old name